jarbas/frontend/elm/Reimbursement/Update.elm

```diff
@@ -414,34 +407,26 @@ loadReimbursements lang apiKey query =
             jsonQuery =
                 ( "format", "json" ) :: convertQuery query
         in
-            Http.get
-                (url "/api/reimbursement/" jsonQuery)
+            Http.get (url "/api/reimbursement/" jsonQuery)
                 (decoder lang apiKey jsonQuery)
                 |> Http.send LoadReimbursements
 
 
 {-| Convert a list of key/value query pairs to a valid URL:
 
-    >>> toUrl [ ( "year", "2016" ), ( "foo", "bar" ) ]
-    "#/year/2016"
-
-    >>> toUrl [ ( "foo", "bar" ) ]
```

Comment:
as fair as I remember, there was no way in the code that this function would have been called with an invalid value, or that wouldn't be a problem, I don't remember now. In fact, maybe we couldn't guarantee that just by looking at this function, maybe the ideal would be having an integration test.

But I'll look again and get back here.